app/models/course/discussion/topic.rb

```diff
@@ -10,6 +10,17 @@ class Course::Discussion::Topic < ActiveRecord::Base
 
   accepts_nested_attributes_for :posts
 
+  # Topics to be displayed in the comments center.
+  scope :to_display, (lambda do
```

Comment:
@lowjoel Thanks ! Just to check, this depends on the specific models all be preloaded right ?  that might not be true in some cases ?